test/suites/warp-sync/test_warp_sync.ts

```diff
@@ -249,11 +249,10 @@ async function directoryExists(directoryPath) {
 
 /// Returns the /tmp/zombie-52234... path
 function getTmpZombiePath() {
-    const logFilePath = process.env.MOON_MONITORED_NODE;
+    const zombiePath = process.env.MOON_ZOMBIE_DIR;
```

Comment:
This function is now useless (and that's good)

```suggestion
    return process.env.MOON_ZOMBIE_DIR;
```